### PR TITLE
added ember snippets

### PR DESCRIPTION
**Pull request by donaldwasserman**
Hey - 

I just started using atom and came across this package (which makes looking at my ember templates a lot easier). 

I wanted to be able to share this with some of my coworkers and along with some additional ember-specific handlebars snippets that I've been using in Sublime.

If you're not interested in this, no big deal, we'll just use the fork.

Thanks for the neat project!

-DW

### CODE REVIEW

File: snippets/language-ember-htmlbars.cson [hunk 1]
```diff
@@ -1 +1,28 @@
...
+    'body': '{{#if ${1:condition}}}\n\t$0\n{{else}}\n\t\n{{/if}}'
+  'unless':
+    'prefix': 'unl'
+    'body': '{{#unless ${1:condition}}}\n\t$0\n{{/unless}}'
+  'ember-action':
+    'prefix': 'ema'
+    'body': '{{action ${1:\'${2:name}\'}}}'
+  'inline-if':
+    'prefix': 'ifl'
+    'body': '{{if ${1:condition} \'${2:property}\'}'
```
Comment (jmurphyau) [inline snippets/language-ember-htmlbars.cson]: Missing a closing }

File: snippets/language-ember-htmlbars.cson [hunk 2]
```diff
@@ -1 +1,28 @@
...
+  'link-to':
+    'prefix': 'lt'
+    'body': '{{#link-to \'${route}\'}}\t$0{{/link-to}}'
```
Comment (jmurphyau) [inline snippets/language-ember-htmlbars.cson:19]: Do you mind adding a inline link-to (similar to inline if - probably as `ltl`)

File: snippets/language-ember-htmlbars.cson [hunk 3]
```diff
@@ -1 +1,28 @@
...
+  'each':
+    'prefix': 'each'
+    'body': '{{#each ${1:model} key="@guid" as |${2:param}|}}\n\t$0\n{{/each}}'
```
Comment (jmurphyau) [inline snippets/language-ember-htmlbars.cson]: Do you mind removing key="@guid"? [This PR](https://github.com/emberjs/ember.js/pull/11461) has made it redundant.

**Comment by jmurphyau** [comment]
Thanks for this.. I removed all the HTML snippets because I wanted HTMLBars ones - just never got around to doing any. 

I've left some inline comments.

**Comment by donaldwasserman** [comment]
Are you responsible for removing that `key=@GUID` syntax? If so, I should be thanking you. That's the reason I started looking for a set of snippets to begin with!

**Comment by schickm** [comment]
Are there plans to merge this pull request?  I'd just go with @donaldwasserman :snippets branch, but he seems to have deleted his fork :cry:

**Comment by donaldwasserman** [comment]
Hey - I ended up creating my own atom package. 

It's called atom-ember-author-kit.

Not the best name. 

DW

**Comment by schickm** [comment]
Heck yes you did, thanks!

**Pull request closed without merging**